doc/index.rst

```diff
@@ -35,19 +32,41 @@ interface toolkits.
    </div>
    <span class="clear_screenshots"></span>
 
-Matplotlib tries to make easy things easy and hard things possible.  You
-can generate plots, histograms, power spectra, bar charts, errorcharts,
-scatterplots, etc., with just a few lines of code.  For examples, see the
-:doc:`sample plots <tutorials/introductory/sample_plots>` and :doc:`thumbnail
-gallery <gallery/index>`.
+Matplotlib makes easy things easy and hard things possible.
 
-For simple plotting the :mod:`.pyplot` module provides a MATLAB-like interface,
-particularly when combined with IPython.  For the power user, you have full
-control of line styles, font properties, axes properties, etc, via an object
-oriented interface or via a set of functions familiar to MATLAB users.
+.. container:: bullet-box-container
+
+   .. container:: bullet-box
+
+      Create
+
+      - Easy creation of standard plots
+      - Publication quality output to common file formats
+        (png, pdf, svg, ps, ...)
+      - Interactive figures (zoom, pan, ...)
+
+   .. container:: bullet-box
+
+      Customize
+
+      - Adapt and tune all aspects a graphic
+      - Embed in interactive environments (e.g. IPython_, Jupyter_)
+      - Create graphics that respond to user GUI events
```

Comment:
My reasoning here is to have the sentences flow under the subheading, and to link out to docs so this acts like a navigation hint of sorts - like these links are why these docs are telling you these things. The GUI stuff is already mentioned under `create`. Also stealing some copy that's already on the front page. 
```
- [Take full control](https://matplotlib.org/tutorials/index.html#tutorials) of line styles, font properties, axes properties...
- [Export and embed](https://matplotlib.org/api/index_backend_api.html) to a number of file formats and interactive environments
```
sorry not sure how these get formatted in rst form

Reply:
ok